src/views/Run/internal/ResultDisplay/TestResult/TestResult.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react'
+
+import { TestPredicate } from '@/benchttp/tests'
+
+import { Accordion } from '../../../../../components/Accordion'
+import { TestResultContent, TestResultTitle } from './internal/components'
```

Comment:
Suggestion, feel free to ignore:

```suggestion
import { TestResultContent } from './TestResultContent'
import { TestResultTitle } from './TestResultTitle'
```

To avoid deeply nested "components.ts" files, we keep the added-value of namespacing at high level.